relay-filter/src/browser_extensions.rs

```diff
@@ -293,4 +324,19 @@ mod tests {
             )
         }
     }
+
+    #[test]
+    fn test_filter_ios_specific_extensions() {
+        let exceptions = ["undefined is not an object (evaluating 'a.L')"];
```

Comment:
The other tests have something like:

```suggestion
        let exceptions = [get_event_with_exception_value_and_os_name("undefined is not an object (evaluating 'a.L')", "iOS");];
```

Could do the same here.